vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2728,6 +2848,75 @@ def create_attn_groups(
                 "All or none of the layers are expected to be encoder-only"
             self.is_encoder_only_model = True
 
+    def initialize_cudagraph_capture(self) -> None:
+        min_cg_support = AttentionCGSupport.ALWAYS
```

Comment:
I think this logic is a bit hard to follow (looks correct), just putting comment down to remember TODO to add better comments